app/view/twig/_macro/_macro.twig

```diff
@@ -15,11 +15,14 @@
 {% macro contentlink(contenttype, content) %}
     {% import _self as crosslinks %}
 
+    {% set title = content.getTitle() %}
+    {#
     {% if content.title is defined %}
         {% set title = content.title %}
     {% else %}
         {% set title = content.name %}
     {% endif %}
+    #}
     {{ crosslinks.contentlink_by_id(contenttype, title, content.id) }}
 {% endmacro %}
```

Comment:
You can just use `content.title`, twig will call `getTitle()`. No need to even set a variable.